R/adverb-auto-browse.R

```diff
@@ -25,7 +25,7 @@
 #'
 auto_browse <- function(.f) {
   if (is_primitive(.f)) {
-    abort("Can not auto_browse() primitive functions")
+    cli::cli_abort("Can not {.fn auto_browse} primitive functions")
```

Comment:
No need to repeat the function name.
Use "can't".
Use full stop.

```suggestion
    cli::cli_abort("Can't browse primitive functions.")
```